daemon/cmd/daemon_main.go

```diff
@@ -2059,19 +2055,6 @@ func (d *Daemon) instantiateAPI() *restapi.CiliumAPIAPI {
 	return restAPI
 }
 
-func initSockmapOption() {
```

Comment:
Thanks, you need to also remove the call to that function (see GitHub Action linter errors).